js/config/syllables.js

```diff
@@ -17,6 +18,8 @@ module.exports = function( locale ) {
 			return de;
 		case "nl":
 			return nl;
+		case "it":
+			return it;
```

Comment:
It would put the languages in an object, because this `switch` statement will grow by each language that will be added.

```js
let languages = { "de" : de, "nl" : nl, "en" : en, "it" : it };

if( languages.hasOwnProperty( getLanguage( locale ) ) ) {
   return languages[ locale ];
}

return languages["en"];
```